packages/grid/rathole/start.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+MODE=${MODE:-server}
+RUST_LOG=${LOG_LEVEL:-trace}
+
+# Copy configuration files
+copy_config() {
+  cp -L -r -f /conf/* conf/
+}
+
+# Start the server
+start_server() {
+  RUST_LOG=$RUST_LOG /app/rathole conf/server.toml &
+}
+
+# Start the client
+start_client() {
+  while true; do
+    RUST_LOG=$RUST_LOG /app/rathole conf/client.toml
+    status=$?
+    if [ $status -eq 0 ]; then
+      break
+    else
+      echo "Failed to load client.toml, retrying in 5 seconds..."
+      sleep 10
+    fi
+  done &
+}
+
+# Reload configuration every 10 seconds
+reload_config() {
```

Comment:
https://github.com/rapiz1/rathole/issues/359 has a PR to fix this symlink issue but not merged yet by project owner

Reply:
Yes sure. We can use `rsync` once this symlink issue is fixed.